include/modem/lte_lc.h

```diff
@@ -1345,18 +1345,85 @@ int lte_lc_normal(void);
  * Requested periodic TAU (RPTAU) and requested active time (RAT) are used when PSM mode is
  * subsequently enabled using lte_lc_psm_req().
  *
- * For reference see 3GPP 27.007 Ch. 7.38.
+ * For encoding of the variables, see nRF AT commands Reference Guide, 3GPP 27.007 Ch. 7.38., and
+ * 3GPP 24.008 Ch. 10.5.7.4a and Ch. 10.5.7.3.
  *
- * @param[in] rptau Requested periodic TAU as a null-terminated string.
+ * @param[in] rptau Requested periodic TAU as a null-terminated 8 character long bit field string.
  *                  Set to @c NULL to use manufacturer-specific default value.
+ *                  See 3GPP 24.008 Ch. 10.5.7.4a for data format.
+ *
+ *                  Value of 12600s is represented as '00101001'.
+ *
+ *                  Bits 5 to 1 represent the binary coded timer value which is
+ *                  multiplied by timer unit.
+ *
+ *                  Bits 8 to 6 defines the timer value unit as follows:
+ *
+ *                  - 000: 10 minutes
+ *                  - 001: 1 hour
+ *                  - 010: 10 hours
+ *                  - 011: 2 seconds
+ *                  - 100: 30 seconds
+ *                  - 101: 1 minute
+ *                  - 110: 320 hours
+ *                  - 111: Timer is deactivated
+ *
  * @param[in] rat Requested active time as a null-terminated string.
  *                Set to @c NULL to use manufacturer-specific default value.
+ *                See 3GPP 24.008 Ch. 10.5.7.3 for data format.
+ *
+ *                Value of 120s is represented as '00100010'.
+ *
+ *                Bits 5 to 1 represent the binary coded timer value which is
+ *                multiplied by timer unit.
+ *
+ *                Bits 8 to 6 defines the timer value unit as follows:
+ *
+ *                - 000: 2 seconds
+ *                - 001: 1 minute
+ *                - 010: 6 minutes
+ *                - 111: Timer is deactivated
  *
  * @retval 0 if successful.
  * @retval -EINVAL if an input parameter was invalid.
  */
 int lte_lc_psm_param_set(const char *rptau, const char *rat);
 
+/**
+ * Set modem PSM parameters.
+ *
+ * Requested periodic TAU (RPTAU) and requested active time (RAT) are used when PSM mode is
+ * subsequently enabled using lte_lc_psm_req().
+ *
+ * This is a convenience function to set PSM parameters in seconds while lte_lc_psm_param_set()
+ * requires the caller to encode the values. However, notice that the underlying AT commands do
+ * not support all possible values but their encoding limits the potential values.
+ * The values are rounded up to the next possible value. Be aware that there may be significant
+ * rounding up especially when rptau is tens and hundreds of hours.
+ * More information about the encodings can be found from lte_lc_psm_param_set().
+ *
+ * @param[in] rptau Requested periodic TAU in seconds as a positive integer. Range 1 - 35711999s.
```

Comment:
Why the upper limit is not 35712000? The lower limit could be 2 seconds, because that's the smallest number that can be encoded.

Reply:
Yep, should be 35712000.